.travis.yml

```diff
@@ -50,7 +50,8 @@ matrix:
         - python: 3.6
           env: TOXENV=py36 SODIUM_INSTALL=system CC=clang
         - env: TOXENV=pypy SODIUM_INSTALL=system CC=clang
-        - env: TOXENV=docs
+        - python: 3.6
```

Comment:
What's going on here?

Reply:
After removing the explicit ``basepython...`` declaration from tox.ini, we add an explicit requirement for a python3.6 runtime in travis docs build stanza. This is needed to avoid getting spurious errors generated by the representation of 2.7 ``str`` vs 3.x ``bytes``

Reply:
👍